pkg/system/chtimes.go

```diff
@@ -8,8 +8,8 @@ import (
 // Chtimes changes the access time and modified time of a file at the given path
 func Chtimes(name string, atime time.Time, mtime time.Time) error {
 	unixMinTime := time.Unix(0, 0)
-	// The max Unix time is 33 bits set
-	unixMaxTime := unixMinTime.Add((1<<33 - 1) * time.Second)
+	// The max Unix time is 63 bits set
+	unixMaxTime := time.Unix(0, 1<<63-1)
```

Comment:
The second parameter of time.Unix(secs, nsecs) is for nanoseconds. I don't know if this is a valid timestamp with 0 seconds + a too big number for the nanoseconds.